packages/calypso-products/src/constants/akismet.ts

```diff
@@ -18,71 +18,86 @@ export const PRODUCT_AKISMET_ENTERPRISE_2M_MONTHLY = 'ak_ep2m_monthly_1';
 export const PRODUCT_AKISMET_ENTERPRISE_GT2M_YEARLY = 'ak_epgt2m_yearly_1';
 export const PRODUCT_AKISMET_ENTERPRISE_GT2M_MONTHLY = 'ak_epgt2m_monthly_1';
 
+export const AKISMET_PERSONAL_MONTHLY_PRODUCTS = <const>[ PRODUCT_AKISMET_PERSONAL_MONTHLY ];
+
 export const AKISTMET_PLUS_MONTHLY_PRODUCTS = <const>[
 	PRODUCT_AKISMET_PLUS_MONTHLY,
 	PRODUCT_AKISMET_PLUS_20K_MONTHLY,
 	PRODUCT_AKISMET_PLUS_30K_MONTHLY,
 	PRODUCT_AKISMET_PLUS_40K_MONTHLY,
 ];
 
+export const AKISMET_BUSINESS_MONTHLY_PRODUCTS = <const>[ PRODUCT_AKISMET_ENTERPRISE_MONTHLY ];
+
 export const AKISMET_ENTERPRISE_MONTHLY_PRODUCTS = <const>[
-	PRODUCT_AKISMET_ENTERPRISE_2M_YEARLY,
+	PRODUCT_AKISMET_ENTERPRISE_350K_MONTHLY,
 	PRODUCT_AKISMET_ENTERPRISE_2M_MONTHLY,
-	PRODUCT_AKISMET_ENTERPRISE_GT2M_YEARLY,
 	PRODUCT_AKISMET_ENTERPRISE_GT2M_MONTHLY,
 ];
```

Comment:
This previously had a YEARLY product in it, and it was specifically meant for MONTHLY products, went ahead and fixed this up